plugins/catalog/src/plugin.ts

```diff
@@ -59,3 +60,36 @@ export const CatalogEntityPage = catalogPlugin.provide(
     mountPoint: entityRouteRef,
   }),
 );
+
+export const EntityHasSystemsCard = catalogPlugin.provide(
+  createComponentExtension({
+    component: {
+      lazy: () =>
+        import('./components/HasSystemsCard/HasSystemsCard').then(
```

Comment:
Could prolly skip the last path component here right? using just `./components/HasSystemsCard` to keep it short :p

Reply:
Would need a index.ts in that folder, but personally would prefer having one. I think I just keept this pattern because the `EntityPage` is doing it above 😆

Reply:
Ah alright xP

Yeah it's usually best to have an index file that exports only the things that should be usable outside, since sometimes you want to export things for tests that you may not want to be used